packages/keyv/src/index.js

```diff
@@ -14,8 +14,12 @@ class Keyv extends EventEmitter {
 			},
 			options
 		);
+		if (options.namespace === false || options.namespace === null || options.namespace === '') {
+			this.options.namespace = undefined;
```

Comment:
why is this necessary?

Reply:
Its not, I just added it as accepted values, any value which is false in Object.assign should work as well

Reply:
pits set to false, then strict checking will not work in `(this.namespace === undefined)` which is used in redis clear

Reply:
We can still flushall if namespace doesn't exist?